modules/claims_api/app/workers/claims_api/claim_establisher.rb

```diff
@@ -20,7 +20,7 @@ def perform(auto_claim_id)
       response = service(auth_headers).submit_form526(form_data)
       auto_claim.evss_id = response.claim_id
       auto_claim.status = ClaimsApi::AutoEstablishedClaim::ESTABLISHED
-      auto_claim.save
+      auto_claim.save!
```

Comment:
I changed this so at least the background job would fail out at this point. Currently, in sandbox, this just returns false and moves on.